src/bench/crypto_hash.cpp

```diff
@@ -105,6 +106,70 @@ static void FastRandom_1bit(benchmark::Bench& bench)
     });
 }
 
+static void MuHash(benchmark::Bench& bench)
+{
+    FastRandomContext rng(true);
+    MuHash3072 acc;
+    unsigned char key[32] = {0};
+    int i = 0;
+    bench.run([&] {
+        key[0] = ++i;
+        acc *= MuHash3072(key);
+    });
+}
+
+static void MuHashAdd(benchmark::Bench& bench)
```

Comment:
Change this to `MuHashMul()`?

Reply:
"multiply" and "divide" (or mul/div) for the maths, and "insert" and "remove" for higher level (multi)set-like operations might make sense. "Add" definitely seems confusing.

Reply:
My preferred naming for the operations is here: https://github.com/bitcoin/bitcoin/pull/19055#discussion_r510232044 (mul/div for Num3072, add/sub for MuHash), but I mostly just want it to be consistent.

Reply:
Done